tutor/resources/styles/components/paging-navigation.less

```diff
@@ -0,0 +1,31 @@
+.tutor-paging-navigation {
+  .flex-display();
+  .justify-content(center);
+  .paging-control {
+    .flex(1);
+    .flex-display();
+    .justify-content(center);
+    .align-items(center);
+    min-height: calc(~"100vh - 150px");
+    .tutor-icon {
+      z-index: 1; //appear above content when page is extremely narrow
+      position: fixed;
+      top: 50vh;
+      color: @tutor-neutral-light;
+      font-size: 150px;
+      .transition(color 150ms linear);
+    }
+    &[disabled] {
+      cursor: not-allowed;
```

Comment:
I can see this being slightly confusing to a user.  To the user, they are hovering their mouse over the side of the page where it looks like a normal background/a continuation of the rest of the page, but they see a not-allowed cursor.